movie/source.rb

```diff
@@ -0,0 +1,17 @@
+require_relative '../item'
+
+class Source < Item
+  attr_reader :id
+  attr_accessor :name, :items
+
+  def initialize(name = '')
+    @id = Random.rand(1..100)
+    @name = name
+    @items = []
+  end
+
+  def add_source(item)
+    @items.push(item)
+    item.source = self
+  end
+end
```

Comment:
As the project requirements, the Source class should not inherit from the Item class in this way. Instead, it would be best if you used the super() method to call the parent class's constructor.